examples/json-record-map-conversion/json_record_map_conversion.bal

```diff
@@ -50,6 +56,6 @@ public function main() {
             age: 30
         }
     };
-    Movie inception = check <Movie>inceptionJ;
+    Movie|error inception = Movie.convert(inceptionJ);
     io:println(inceptionJ);
```

Comment:
Don't we need to check the type before printing the value? We do that in other places above.